Automattic/isolated-block-editor
# Collab: Fix peer caret glitches at list item edges

Fixes #48 
Based on #104 

Adds special handling for when a peer caret is at the ending edge of a List block item.

## Testing instructions

1. `yarn storybook`
2. Open two browser tabs at http://localhost:6006/?path=/story/collaboration--default
3. See repro steps in the original issue (#48)

Note: A peer caret will not show when it is at the beginning of an empty line. This is an existing limitation.
